iscript/src/iscript/mac.py

```diff
@@ -377,7 +377,7 @@ def get_app_dir(parent_dir):
         UnknownAppDir: if there is no single app dir
 
     """
-    apps = glob("{}/*.app*".format(parent_dir)) + glob("{}/*/*.app*".format(parent_dir))
+    apps = glob("{}/*.app*".format(parent_dir)) + glob("{}/*/*.app*".format(parent_dir)) + glob("{}/*.systemextension*".format(parent_dir))
```

Comment:
I'm ok with this. I could also see `iscript.constants.MAC_PRODUCT_CONFIG.PRODUCT.app_globs` or `app_extensions` if we need to customize this anymore :)